iot-hub/Samples/service/RoleBasedAuthenticationSample/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Azure.Identity;
+using CommandLine;
+using Microsoft.Azure.Devices;
+using System;
+using System.Threading.Tasks;
+
+namespace RoleBasedAuthenticationSample
+{
+    public class Program
+    {
+        /// <summary>
+        /// A sample to illustrate how to use Azure active directory for authentication to the IoT hub.
```

Comment:
will you be adding readme for this sample?

Reply:
Can you explain? Do we keep some readme somewhere?

Reply:
Chatted offline. Will send a follow up PR with readme.